src/nvim/ex_docmd.c

```diff
@@ -6426,6 +6426,19 @@ ex_win_close(
   int need_hide;
   buf_T       *buf = win->w_buffer;
 
+  if (!tp) {
+    tp = curtab;
+  }
+
+  FOR_ALL_WINDOWS_IN_TAB(wp, tp)  {
+    if (wp->w_floating) {
+      if (wp->w_float_config.relative == kFloatRelativeWindow
+          && wp->w_float_config.window == win->handle) {
+        ex_win_close(forceit, wp, tp);  
```

Comment:
`tp` here must still be NULL when it is the current tab.

Reply:
I did what you said, but that did not fix the error. 

Upon further debugging I found that the `w_buffer` for `win` is NULL in the `hasAnyFolding` function.

Reply:
It wasn't about that particular error (necessarily). It still needs to be done.

Reply:
Done! :)